promoted_python_delivery_client/model/user_info.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass, field
+from dataclasses_json import config, dataclass_json, LetterCase
+from typing import Optional
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class UserInfo:
+    user_id: Optional[str] = field(default=None, metadata=config(exclude=lambda v: v is None))  # type: ignore
+    log_user_id: Optional[str] = field(default=None, metadata=config(exclude=lambda v: v is None))  # type: ignore
+    is_internal_user: Optional[bool] = field(default=None, metadata=config(exclude=lambda v: v is None))  # type: ignore
```

Comment:
Side - I added an `ignore_usage` about 1-2 months ago.  Not urgent to add.

Reply:
Ok, I will also add that in Java.